OpenEdXMobile/src/main/java/org/edx/mobile/view/MainDashboardActivity.java

```diff
@@ -55,7 +56,7 @@ public static Intent newIntent() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        NotificationUtil.subscribeToTopics(environment);
```

Comment:
@farhan @miankhalid Where is the right place to put this so that it will get triggered when a user opens to application?

I tried moving it to the SplashActivity, but that is not created every time the app is opened fresh (not from the background).